crates/rullm-chat-completion/spec/chat-completion-comparison.md

```
@@ -0,0 +1,379 @@
+| Tool call | `tool_calls` | `tool_use` | (function call in content) |
+| Content filter | `content_filter` | — | `SAFETY` |
+| Copyright | — | — | `RECITATION` |
+
+**Gemini-specific**: `RECITATION` triggers when output is too similar to copyrighted training data, blocking the response.
+
+## Streaming
+
+All providers use Server-Sent Events (SSE), but event structure differs significantly.
+
+**OpenAI/Groq streaming chunk:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)